analyzer/src/main/kotlin/managers/CocoaPods.kt

```diff
@@ -169,7 +169,7 @@ class CocoaPods(
     }
 
     private fun getPackage(id: Identifier, workingDir: File): Package {
-        val podspec = getPodspec(id, workingDir)
+        val podspec = getPodspec(id, workingDir) ?: return Package.EMPTY.copy(id = id)
```

Comment:
Shouldn't this at least also emit a warning / hint?

Reply:
I made this a TODO for now, as I wanted to eliminate the NPE quickly.